HappyAnding/HappyAnding/Views/Components/ShortcutCell.swift

```diff
@@ -110,8 +110,8 @@ struct ShortcutCell: View {
     var shortcutInfo: some View {
         
         VStack(alignment: .leading, spacing: 0) {
-            if rankNumber != -1 {
-                Text("\(rankNumber)")
+            if (rankNumber != nil) {
+                Text(String(rankNumber! + 1))
```

Comment:
```suggestion
            if let rankNumber {
                Text(String(rankNumber + 1))
```

옵셔널 바인딩 사용해서 강제 언래핑을 사용하지 않는 건 어떨까요!?

Reply:
이게 훨씬 안전하겠네요! 반영하겠습니당